apps/sasquatch/src/projectDependency/res/values/array.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string-array name="appcenter_secrets" tools:ignore="MissingTranslation">
+        <item>9e0d97c1-7838-46d0-9dab-1a0ef66aec6e</item>
```

Comment:
<img width="698" alt="Screen Shot 2019-09-09 at 2 12 04 PM" src="https://user-images.githubusercontent.com/623672/64566948-f9c44000-d30b-11e9-89f0-a46a9e6f005a.png">

Same for the other variant.

Also please move this key to `env.xml` even if that one is an array, the files are actually not typed. `env.xml` is more convenient to have all the settings in 1 place.

Reply:
Thanks. I've moved to env.xml, and it looks like now that it's in env.xml, TypographyDashes is already in the ignore list due to resources element having it.